packages/scale/index.d.ts

```diff
@@ -0,0 +1,130 @@
+/**
+ * Get scale properties. It returns an object with:
+ * - name: the scale name
+ * - names: a list with all possible names (includes the current)
+ * - intervals: an array with the scale intervals
+ * - chroma:  scale croma (see pcset)
+ * - setnum: scale chroma number
+ *
+ * @function
+ * @param {String} name - the scale name (without tonic)
+ * @return {Object}
+ */
+export declare const props: (
+  name: string
+) => {
+  name: string;
+  names: string[];
+  intervals: string[];
+  chroma: string;
+  setnum: string;
+};
+/**
+ * Return the available scale names
+ *
+ * @function
+ * @param {boolean} aliases - true to include aliases
+ * @return {Array} the scale names
+ *
+ * @example
+ * Scale.names() // => ["maj7", ...]
+ */
+export declare const names: (aliases: boolean) => string[];
+/**
+ * Given a scale name, return its intervals. The name can be the type and
+ * optionally the tonic (which is ignored)
+ *
+ * It retruns an empty array when no scale found
+ *
+ * @function
+ * @param {String} name - the scale name (tonic and type, tonic is optional)
+ * @return {Array<String>} the scale intervals if is a known scale or an empty
+ * array if no scale found
+ * @example
+ * Scale.intervals("major") // => [ "1P", "2M", "3M", "4P", "5P", "6M", "7M" ]
+ */
+export declare const intervals: (name: string) => string[];
+/**
+ * Get the notes (pitch classes) of a scale.
+ *
+ * Note that it always returns an array, and the values are only pitch classes.
+ *
+ * @function
+ * @param {String} nameOrTonic - the scale name or tonic (if 2nd param)
+ * @param {String} [name] - the scale name without tonic
+ * @return {Array} a pitch classes array
+ *
+ * @example
+ * Scale.notes("C", "major") // => [ "C", "D", "E", "F", "G", "A", "B" ]
+ * Scale.notes("C major") // => [ "C", "D", "E", "F", "G", "A", "B" ]
+ * Scale.notes("C4", "major") // => [ "C", "D", "E", "F", "G", "A", "B" ]
+ * Scale.notes("A4", "no-scale") // => []
+ * Scale.notes("blah", "major") // => []
+ */
+export declare function notes(nameOrTonic: string, name: string): string[];
```

Comment:
I'd love to have something like this:

```ts
export declare function notes(nameOrTonic: Note | ScaleName, name?: ScaleName): string[];
```

Anyway, the second parameter is optional:
```suggestion
export declare function notes(nameOrTonic: string, name?: string): string[];
```